### PR TITLE
Removed methods and variables with unused annotation

### PR DESCRIPTION
**Pull request by fcalderonnearsoft**
I found more unused methods and variables related with  https://github.com/Netflix/Hystrix/issues/1150

### CODE REVIEW

**Comment by fcalderonnearsoft** [comment]
Hi @mattrjacobs when you have time could you check this, please?

File: hystrix-contrib/hystrix-metrics-event-stream/src/main/java/com/netflix/hystrix/contrib/metrics/eventstream/HystrixMetricsPoller.java [hunk 1]
```diff
@@ -134,19 +134,6 @@ public boolean isRunning() {
     /**
      * Used to protect against leaking ExecutorServices and threads if this class is abandoned for GC without shutting down.
      */
-    @SuppressWarnings("unused")
```
Comment (mattrjacobs) [inline hystrix-contrib/hystrix-metrics-event-stream/src/main/java/com/netflix/hystrix/contrib/metrics/eventstream/HystrixMetricsPoller.java]: This one does get called, just not within the Hystrix library.  See http://stackoverflow.com/questions/6872857/how-does-a-finalizer-guardian-work-in-java for a full explanation of how this works.  This one should stay.

**Comment by mattrjacobs** [comment]
@fcalderonnearsoft Sorry for the delay - was out on vacation for a few days.  I left a note on the finalizer guardian, but the others look fine to remove.  If you can re-add the finalizer guardian, I will merge the PR.  Thanks for the contribution!

**Comment by fcalderonnearsoft** [comment]
@mattrjacobs  No problem, I already re-add finalizerguardian in a new commit. 

Cheers.

**Pull request merged**